test/evergreen.yml

```diff
@@ -2207,15 +2187,6 @@ tasks:
       - func: "make check all"
         vars:
           test_env_vars: ${test_env_vars} TESTUTIL_ENABLE_LONG_TESTS=1
-      - command: shell.exec
-        params:
-          working_dir: "wiredtiger/build_posix"
-          script: |
-            set -o errexit
-            set -o verbose
-
-            WT3363_CHECKPOINT_OP_RACES=1 test/csuite/./test_wt3363_checkpoint_op_races 2>&1
```

Comment:
It looks the "wt3363 checkpoint op races" test has been covered by "make check all", though I'm not sure whether `WT3363_CHECKPOINT_OP_RACES=1` setting here would bring additional coverage or not. Any idea?

Reply:
I added some info in the ticket [WT-8504](https://jira.mongodb.org/browse/WT-8504) and had a discussion with @hbokhari, this option no longer exists.

Reply:
Thanks @etienneptl (sorry I did not check the Jira ticket description).